Sources/Brave/Frontend/Browser/TabManagerNavDelegate.swift

```diff
@@ -67,9 +67,16 @@ class TabManagerNavDelegate: NSObject, WKNavigationDelegate {
     }
   }
 
-  private func defaultAllowPolicy() -> WKNavigationActionPolicy {
+  private func defaultAllowPolicy(for navigationAction: WKNavigationAction) -> WKNavigationActionPolicy {
     let isPrivateBrowsing = tabManager?.privateBrowsingManager.isPrivateBrowsing == true
-    if isPrivateBrowsing || !Preferences.General.followUniversalLinks.value {
+    func isYouTubeLoad() -> Bool {
+      guard let domain = navigationAction.request.mainDocumentURL?.baseDomain else {
+        return false
+      }
+      return domain == "youtube.com"
```

Comment:
i think its fine. Can you check if `youtu.be` domains would need to be added as well? I think no because it redirects to the regular domain

Reply:
I thought it would redirect before as well but it looks like YouTube does in fact have universal links set up for `youtu.be` as well, so adding this domain to check too